### ISSUE TITLE
apt-key is deprecated

### ISSUE THREAD

**Issue by fwdekker** [meganz/MEGAsync#683]
When running `apt update` on a Debian system with megasync installed, the following warning is shown:

```
W: https://mega.nz/linux/repo/Debian_11/.InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.
```

Indeed, [megasync's post-install script](https://github.com/meganz/MEGAsync/blob/1a1922c0ff88537764501b7b93c179da444f21dc/build/MEGAsync/MEGAsync/debian.postinst) uses `apt-key` to add the repository's key.

---

Mega should use the replacement storage method, which is [explained on the Debian Wiki](https://wiki.debian.org/DebianRepository/UseThirdParty#OpenPGP_Key_distribution), [explained in this article](https://www.linuxuprising.com/2021/01/apt-key-is-deprecated-how-to-add.html), [explained in this AskUbuntu answer](https://askubuntu.com/a/1307181), [explained in my StackOverflow answer](https://stackoverflow.com/a/71384057), and  [used by Signal](https://signal.org/download/).

In short, instead of running `apt-key add ...`, the key should be dearmored and stored in, say, `/usr/share/keyrings/meganz.gpg`, and the `/etc/apt/sources.list.d/megasync.list` entry should be changed to include `[signed-by=/usr/share/keyrings/meganz.gpg]`. Finally, the old key should be removed using `apt-key del support@mega.co.nz`.

I ran these commands locally and it all seems to work fine.

**Comment by polmr** [comment]
Thanks @FWDekker , we are in the process of addressing such a warning. Thanks for caring to prepare such a well documented issue.

**Comment by FWDekker** [comment]
This has been fixed in 706757d55566bf83007abd17defc4fefa97ace4b.
